server/server.go

```diff
@@ -398,6 +401,18 @@ func NewServer(configFile string) (*Server, error) {
 			SkipLogging: true,
 		}))
 	}
+	etcdKey := config.GetString("healthcheck/etcd_key", "/gohan")
+	var gohanPort int
+	addressAndPort := strings.Split(server.address, ":")
+	if len(addressAndPort) != 2 {
+		panic("Incorrect gohan server address: " + server.address)
+	}
+	gohanPort, err = strconv.Atoi(addressAndPort[1])
+	if err != nil {
+		panic("Incorrect gohan server address: expected port number got " + addressAndPort[1])
+	}
+	healthCheckAddress := config.GetString("healtcheck/address", fmt.Sprintf("%s:%d", addressAndPort[0], gohanPort+1))
+	server.Healthcheck = healthcheck.NewHealthCheck(server.db, server.sync, etcdKey, healthCheckAddress, log)
```

Comment:
Now there's no way to disable the healthcheck. It's an unauthenticated endpoint, so it's a bit serious. I think we should add a configuration knob with disabled as default.